WebApp/main.py

```diff
@@ -1,14 +1,77 @@
-import os
-from fastapi import FastAPI, File, UploadFile
+from fastapi import FastAPI, Depends, HTTPException, status, File, UploadFile
 from fastapi.responses import JSONResponse
-from typing import Union
-from fastapi import FastAPI
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from fastapi import Form
+from sqlalchemy import Column, Integer, String, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, Session
+import bcrypt
+import os
+from pydantic import BaseModel
 
 app = FastAPI()
 
 UPLOAD_DIR = "uploaded_images"
 os.makedirs(UPLOAD_DIR, exist_ok=True)
 
+DATABASE_URL = "sqlite:///./user-data.db"
+Base = declarative_base()
+engine = create_engine(DATABASE_URL, connect_args={"check_same_thread": False})
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+
+# Database Model
+class User(Base):
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True, index=True)
+    username = Column(String, unique=True, index=True)
+    hashed_password = Column(String)
+
```

Comment:
Proponuję dodanie pola `email` do modelu `User`. Jest to istotne dla funkcjonalności:

- Umożliwi wysyłanie powiadomień do użytkowników
- Pozwoli na implementację resetu hasła przez email
- Będzie służyć jako alternatywny identyfikator podczas logowania

Sugerowana zmiana:

```typescript
interface User {
 id: string;
 name: string;
 email: string; // nowe pole
}